packages/nlopt/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://nlopt.readthedocs.io/
+TERMUX_PKG_DESCRIPTION="Library for nonlinear optimization"
+TERMUX_PKG_LICENSE="LGPL-2.1, MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.6.2
```

Comment:
Any particular reason for not packaging the latest release (2.7.0)?

Reply:
Ah, I did not notice that a newer version had been released just the day before.